ReactTask/src/redux/patient/patientOperations.ts

```diff
@@ -55,7 +52,45 @@ export const addPatient =
         values
       );
       dispatch(createPatientSuccess(data));
+      dispatch(
+        alert({
+          show: true,
+          err: false,
+          message: 'Patient added successfully',
+        })
+      );
     } catch (error) {
+      dispatch(
+        alert({
+          show: true,
+          err: false,
+          message: 'Something went wrong!',
+        })
+      );
       dispatch(createPatientError((error as Error).message));
     }
   };
+
+export const getResolutionsPatient =
+  () => async (dispatch: Dispatch<{ type: string }>) => {
+    dispatch(getResolutionsPatientRequest());
+
+    try {
+      const { data } = await axios.get('/api/resolutions/patient/me', {
```

Comment:
By the way /api could be added to baseURl

Reply:
Good idea
Thank you